assets/js/faqs.js

```diff
@@ -4,11 +4,13 @@ const openAccordionCount = {
   development: 0,
 };
 
+const capitalize = s => s && s[0].toUpperCase() + s.slice(1)
+
 function toggleExpandAllCollapseAllBtn(section) {
   const btns = document.getElementsByClassName(`collapse-expand-btn-${section}`);
 
   Array.from(btns).forEach((btn) => {
-    btn.innerHTML = openAccordionCount[section] > 0 ? 'Collapse all' : 'Expand all';
+    btn.innerHTML = openAccordionCount[section] > 0 ? 'Collapse all ' + capitalize(section) + ' FAQ' : 'Expand all ' + capitalize(section) + ' FAQ';
```

Comment:
Maybe we could just replace the "Collapse" / "Expand" portion of the text? That way we don't need to generate a label for the section and can remove the `capitalize` helper.

```js
btn.textContent = btn.textContent.replace(/Collapse|Expand/, openAccordionCount[section] > 0 ? 'Collapse' : 'Expand');
```